client/test/components/conference_room/provider/EventProvider.spec.jsx

```diff
@@ -58,4 +63,46 @@ describe('<EventProvider />', () => {
     defaultWrapper.find(DummyComponent).simulate('update');
     expect(EventSource.fetch).to.have.been.calledOnce();
   });
+
+  describe('reloading page on next day', () => {
+    context('on 00:00 AM', () => {
+      let clock;
+      before(() => {
+        clock = sinon.useFakeTimers();
+      });
+
+      after(() => {
+        clock.restore();
+      });
+
+      it('reloads page on next day', () => {
+        mount(<EventProvider conferenceRoom={conferenceRoom}
+                             component={DummyComponent} {...ownProps} />);
+        EventSource.fetch.reset();
+        clock.tick(1000 * 60 * 60 * 24);
```

Comment:
How about extracting a constant here? `MILLISECONDS_IN_A_DAY`? The same constant is used in line 103